package.json

```diff
@@ -25,16 +25,30 @@
     "@popperjs/core": "^2.11.6",
     "@tanstack/react-query": "^5.14.1",
     "@tanstack/react-table": "^8.7.9",
-    "axios": "^1.3.2",
+    "axios": "^1.8.3",
+    "classnames": "^2.5.1",
     "dompurify": "^3.0.5",
+    "html-to-react": "^1.7.0",
+    "immutability-helper": "^3.1.1",
+    "lodash": "^4.17.21",
     "lodash.truncate": "^4.4.2",
+    "moment": "^2.30.1",
     "prop-types": "^15.8.1",
     "qs": "^6.11.2",
     "react-datepicker": "^4.10.0",
+    "react-dnd-html5-backend": "^16.0.1",
+    "react-dnd": "^16.0.1",
+    "react-perfect-scrollbar": "^1.5.8",
     "react-popper": "^2.3.0",
+    "react-popper-tooltip": "^4.4.2",
     "react-responsive": "^9.0.2",
     "react-router-dom": "^6.8.0",
-    "swagger-ui-react": "^5.11.3"
+    "react-router-hash-link": "^2.4.3",
+    "react-socks": "^2.2.0",
+    "react-table": "^7.8.0",
```

Comment:
We probably shouldn't have both "react-table" and "@tanstack/react-table". If posible we should deprecate the older and update the components that use it